### DISCUSSION TITLE
How can I set up the accelerator button travel on my controller to basically correspond to the button pressure travel on the PS2 platform?

### DISCUSSION THREAD

**Discussion by ekang4887** [darthcloud/BlueRetro#1072] [Q&A]
I have a question. I'm playing Gran Turismo 4 on the PS2 platform using a Google Stadia controller. I've mapped the LT and RT buttons to the brake and accelerator. However, when I press the LT and RT buttons halfway, the car is already at full throttle or full braking. How should I set the L2 and R2 button parameters in BlueRetro to ensure that the travel of my LT and RT buttons fully corresponds to the travel of the L2 and R2 buttons on the PS2 controller?

**Comment by darthcloud** [comment]
Try to change the Max field value to less than 100%
